diff --git a/dist/core/sync/index.js b/dist/core/sync/index.js
index 25571f63045b4ac7a8eac6788f744380b1235e4f..00ffdcb2747e4b1fb08f5feb70ec095db4e69259 100644
--- a/dist/core/sync/index.js
+++ b/dist/core/sync/index.js
@@ -210,7 +210,7 @@ async function syncContentCollections(settings, {
       { cause: e }
     );
   } finally {
-    await tempViteServer.close();
+    // await tempViteServer.close();
   }
 }
 export {
